src/screens/Home.js

```diff
@@ -49,6 +50,17 @@ export default ({
     navigation.navigate('Learn');
   };
 
+  const switchNativeLanguage = () => {
+    setLanguageName(
+      nativeLanguage === LANGUAGE_NAMES.EN
+        ? LANGUAGE_NAMES.MG
+        : LANGUAGE_NAMES.EN,
+    );
+  };
```

Comment:
this is duplicated in home.js
create action toggleLanguage with code above and reuse to become
 const switchNativeLanguage = () => {
    toggleLanguage();
  };
  or just pass toggleLanguage straight to onPress on component